### PR TITLE
fix: Check for split-delegation strategy with delegationOverride

### PR DESCRIPTION
**Pull request by ChaituVR**
### Summary
This PR is to enable vote overriding for `split-delegation` strategy https://github.com/snapshot-labs/snapshot-strategies/pull/1484 If `delegationOverride` is `true`

### How to test:
- Create a proposal by enabling `split-delegation` strategy on your space with `delegationOverride` param as `true`
- User should be able to override their vote

### CODE REVIEW

**Comment by wa0x6e** [comment]
No changes?

File: src/helpers/utils.ts [hunk 1]
```diff
@@ -72,7 +72,16 @@ export function hasStrategyOverride(strategies: any[]) {
...
+  if (keywords.some(keyword => strategiesStr.includes(`"name":${keyword}`))) return true;
+  // Check for split-delegation with delegationOverride
+  const splitDelegation = strategies.filter(strategy => strategy.name === 'split-delegation');
+  if (
+    splitDelegation.length > 0 &&
+    splitDelegation.some(strategy => strategy.params?.delegationOverride)
+  )
+    return true;
+
+  return false;
```
Comment (wa0x6e) [inline src/helpers/utils.ts:84]:
```suggestion
  return (
    splitDelegation.length > 0 &&
    splitDelegation.some(strategy => strategy.params?.delegationOverride)
  )
```

**Review by wa0x6e — APPROVED** [review]

**Comment by wa0x6e** [comment]
utAck, looks good

File: src/helpers/utils.ts [hunk 2]
```diff
@@ -72,7 +72,13 @@ export function hasStrategyOverride(strategies: any[]) {
...
   ];
   const strategiesStr = JSON.stringify(strategies).toLowerCase();
-  return keywords.some(keyword => strategiesStr.includes(`"name":${keyword}`));
+  if (keywords.some(keyword => strategiesStr.includes(`"name":${keyword}`))) return true;
+  // Check for split-delegation with delegationOverride
+  const splitDelegation = strategies.filter(strategy => strategy.name === 'split-delegation');
+  return (
+    splitDelegation.length > 0 &&
+    splitDelegation.some(strategy => strategy.params?.delegationOverride)
+  )
```
Comment (ChaituVR) [inline src/helpers/utils.ts:81]:
```suggestion
  );
```

**Pull request merged**